src/CardCreateDialog.vue

```diff
@@ -84,17 +84,17 @@
 							{{ t('deck', 'Close') }}
 						</button>
 					</template>
-				</EmptyContent>
+				</NcEmptyContent>
 			</div>
 		</div>
-	</Modal>
+	</NcModal>
 </template>
 
 <script>
 import { generateUrl } from '@nextcloud/router'
-import Modal from '@nextcloud/vue/dist/Components/Modal'
-import Multiselect from '@nextcloud/vue/dist/Components/Multiselect'
-import EmptyContent from '@nextcloud/vue/dist/Components/EmptyContent'
+import NcModal from '@nextcloud/vue/dist/Components/Modal'
```

Comment:
The import file probably needs changing too, right?

Reply:
Good catch, missed that in my regex replacing